docs/commands.md

```diff
@@ -4652,6 +4652,7 @@ hal config provider azure account add ACCOUNT [parameters]
  * `--subscription-id`: (*Required*) The subscriptionId that your service principal is assigned to.
  * `--tenant-id`: (*Required*) The tenantId that your service principal is assigned to.
  * `--write-permissions`: (*Default*: `[]`) A user must have at least one of these roles in order to make changes to this account's cloud resources.
+ * `--regions`: (*Default*: `['westus', 'eastus']`) The Azure regions this Spinnaker account will manage.
```

Comment:
This looks to be edited by hand. This file is auto-generated by running these commands: https://github.com/spinnaker/halyard/blob/master/halyard-cli/Makefile#L2-L3

Your hand edits will be overwritten the next time somebody generates this file, so please auto-generate it as part of this PR.

When you do that, you'll see that the doc generator will include the actual default value of the parameter in the description (in this case, and empty list).